server/server.js

```diff
@@ -185,6 +183,21 @@ Server.prototype._resourceGet = function(name) {
   return this.resources[name];
 };
 
+// Subscribe to the persistence pubsub channel for a single resource
+Server.prototype._persistenceSubscribe = function (name, id) {
+  if (!this.subs[name]) {
+    logging.info('#redis - subscribe', name, id);
```

Comment:
Think you can change these two info's into debugs. Not strictly needed for debugging.